lms/envs/common.py

```diff
@@ -1247,7 +1247,8 @@
     'js/vendor/requirejs/require.js',
     'js/RequireJS-namespace-undefine.js',
     'js/vendor/URI.min.js',
-    'js/vendor/backbone-min.js'
+    'js/vendor/backbone-min.js',
+    'js/vendor/afontgarde/modernizr.fontface-generatedcontent.js',
```

Comment:
OK after removing this instance of Modernizr, the icon fonts don't have their accessible fallbacks. Since this PR needs to get in and done, I'm going to include this version for now. Whenever the UXPL is integrated into the platform, we can use that version of Modernizr instead of this one.